### PR TITLE
Add unit test

### PR DESCRIPTION
**Pull request by andriihalai**

### CODE REVIEW

File: src/routes/auth/refresh/server.test.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      cookies: {
+        set: vi.fn(),
+        delete: vi.fn(),
+      },
+    };
+
+    const response = await POST(mE);
+
+    expect(mE.request.json).toHaveBeenCalled();
+    expect(mE.cookies.set).toHaveBeenCalledWith(COOKIE_SESSION_NAME, 'test-token', { path: '/' });
```
Comment (KseniiaHorai) [inline src/routes/auth/refresh/server.test.ts:26]: This line is too long. Make it shorter. 
```suggestion
    expect(mE.cookies.set).toHaveBeenCalledWith(COOKIE_SESSION_NAME, 'test-token', { path: '/' });
```

File: src/routes/auth/refresh/server.test.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { COOKIE_SESSION_NAME } from '../constants';
+import { POST } from './+server';
+
+vi.mock('@sveltejs/kit', () => ({
+  json: vi.fn((data) => data),
+}));
+
+describe('POST function', () => {
+  it('should set the cookie if token is provided', async () => {
+    const mE = {
```
Comment (KseniiaHorai) [inline src/routes/auth/refresh/server.test.ts]: Change the name of the variable.
```suggestion
    const mockEvent = {
```

File: src/routes/auth/refresh/server.test.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        set: vi.fn(),
+        delete: vi.fn(),
+      },
+    };
+
+    const response = await POST(mE);
+
+    expect(mE.request.json).toHaveBeenCalled();
+    expect(mE.cookies.set).not.toHaveBeenCalled();
+    expect(mE.cookies.delete).toHaveBeenCalledWith(COOKIE_SESSION_NAME, { path: '/' });
```
Comment (KseniiaHorai) [inline src/routes/auth/refresh/server.test.ts]: This line is too long. Make it shorter.

File: src/routes/auth/refresh/server.test.ts (hunk 1 above)
Comment (KseniiaHorai) [inline src/routes/auth/refresh/server.test.ts]: This line is too long. Make it shorter.

File: src/routes/(authed)/layout.server.test.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+vi.mock('@sveltejs/kit', () => ({
+  redirect: vi.fn().mockImplementation(() => {
+    const error = new Error('Redirect');
+    // eslint-disable-next-line
+    (error as any).status = 303;
+    // eslint-disable-next-line
+    (error as any).location = '/auth';
+    throw error;
+  }),
+}));
```
Comment (erotourtes) [inline src/routes/(authed)/layout.server.test.ts:14]: You might try to use the built-in `redirect` function (there is `error` fn as well).  Take a look https://learn.svelte.dev/tutorial/redirects

File: src/routes/(authed)/layout.server.test.ts [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+describe('load', () => {
+  it('should redirect to /auth if user is not present', async () => {
+    const mockEvent = { locals: { user: null } };
+
+    try {
+      await load(mockEvent);
+    } catch (error) {
+      console.log('Error here');
```
Comment (erotourtes) [inline src/routes/(authed)/layout.server.test.ts]: It is unnecessary, but it's better not to bloat code with `console.log`s

Also, recently, I found out this about construction `await expect(() => throwFn()).rejects.to...`, but your code is solid as well

File: src/routes/(authed)/layout.server.test.ts (hunk 4 above)
Comment (andriihalai) [reply src/routes/(authed)/layout.server.test.ts:14]: I get an error for some reason when i try to do it

**Comment by erotourtes** [comment]
:shipit:

**Pull request merged**